team.php

```diff
@@ -118,7 +118,83 @@ function team() {
 		}
 		$system->printMessage();
 	}
-	else if($_GET['join_mission'] && $player->team['mission_id']) {
+	else if (isset($_POST['set_boost'], $_POST['set_amount'])) {
+
+		$boost_valid = true;
+		$Boost = $system->clean($_POST['set_boost']);

```

Comment:
Is the intent of the capitalized first letter here to denote a variable derived from input (and maybe thus that should not be modified)? If so I think maybe `$_boost` would be a good way to set this

Reply:
Not really a deep reason. It was late at night so I just randomly did it lol

Reply:
Haha gotcha